libs/service-portal/settings/access-control/src/index.ts

```diff
@@ -14,26 +14,29 @@ export const accessControlModule: ServicePortalModule = {
   routes: ({ userInfo }) => {
     const isCompany = userInfo.profile.subjectType === 'legalEntity'
     const isDelegation = Boolean(userInfo.profile.actor)
+    const showDelegation =
+      isCompany || isDelegation
```

Comment:
Why are we checking this?

The scope check should be enough. If it is not correct in some case then that indicates a security issue we need to fix.

Reply:
I agree - This has been done a little bit differently in another PR that has been merged already, so i will delete this PR. 
I'll double check that we can trust the scopes only and refactor these checks.